src/main/java/mate/academy/service/impl/UserServiceImpl.java

```diff
@@ -15,8 +15,9 @@ public class UserServiceImpl implements UserService {
 
     @Override
     public User add(User user) {
-        user.setSalt(HashUtil.getSalt());
-        user.setPassword(HashUtil.hashPassword(user.getPassword(), user.getSalt()));
+        HashUtil hashUtil = HashUtil.getInstance();
```

Comment:
why do you need this variable?

Reply:
Yes, my implementation of HashService uses non static methods

Reply:
but why do you need a non-static method for the util class? 
[See an example of utility here](https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html)